libraries/chain/worker_evaluators.cpp

```diff
@@ -386,15 +388,23 @@ namespace golos { namespace chain {
                 });
             }
         } else if (o.state == worker_techspec_approve_state::approve) {
+            auto month_sec = fc::days(30).to_seconds();
             auto payments_period = int64_t(wto.payments_interval) * wto.payments_count;
 
             uint128_t cost(wto.development_cost.amount.value);
             cost += wto.specification_cost.amount.value;
-            cost *= std::min(fc::days(30).to_seconds(), payments_period);
+            cost *= std::min(month_sec, payments_period);
             cost /= payments_period;
             auto consumption = asset(cost.to_uint64(), STEEM_SYMBOL);
 
-            GOLOS_CHECK_LOGIC((gpo.worker_consumption_per_month + consumption) <= gpo.worker_revenue_per_month,
+            int128_t prognosis_funds(gpo.worker_revenue_per_month.amount.value);
```

Comment:
Rewrite logic to use uint128_t.
uint128_t is a custom implementation in fc-library, and it works.

```
uint128_t revenue_funds = ....
uint128_t consumption_funds = ....

GOLOS_CHECK_LOGIC(revenue_funds >= consumption_funds,  ...
```